src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -195,16 +83,32 @@ void GcsPlacementGroupScheduler::ScheduleUnplacedBundles(
     return;
   }
 
+  RAY_CHECK(bundles.size() == selected_nodes.size());
+
+  // Covert to a map of bundle to node.
```

Comment:
To be clear:

Covert `bundles` and `selected_nodes` to a map of bundle to node, which is the param that `LeaseStatusTracker` depends on.

These 5 lines if the function of original `GenerateScheduleResult`.

I think the original `ScheduleResult` and new data structure `SchedulingResult` are easily confused, so I just remove the original `GenerateScheduleResult` method and use these simple 5 lines to instead of it.